src/datta_lab_to_nwb/markowitz_gillis_nature_2023/markowitz_gillis_nature_2023_metadata.yaml

```diff
@@ -23,13 +23,15 @@ NWBFile:
     - Datta, Sandeep Robert
 Subject:
   species: Mus musculus
+  age: P6W/ # TODO: find DOBs for each animal
+  date_of_birth: 2000-01-01T00:00:00+00:00 # Dummy dob for dandi
```

Comment:
Not that you need both; for `age` vs. `date_of_birth` the only requirement is 'one or the other'